### DISCUSSION TITLE
Cyclic-free graph violated error in TFC for BBN-PYNQ example for 2w2a (weight 2 bits, activation 2bits)

### DISCUSSION THREAD

**Discussion by pkdeep** [Xilinx/finn#937] [Q&A]
Hello I am trying to run BBN-PYNQ example notebook with TFC (for MNIST data) network with weight = 2bits and activation also as 2 bit. I am getting cycle-free graph violated error while generating dataflow partition. Any workaround? 
(docker image:           xilinx/finn                   v0.9-2-gb3bdff11-dirty.xrt_202210.2.13.466_18.04-amd64-xrt)
Below in the screenshot of the error.
![Finn_graph_error](https://github.com/Xilinx/finn/assets/78947824/6040a7d9-0d05-465f-92f7-b7d03f4f1411)

**Comment by dimlol** [comment]
Did you find any solution for this error?
